tests/test_notify_hook.py

```
@@ -316,6 +316,46 @@ def test_get_assistant_messages(self):
+        self.assertEqual(context, "")
+
+    def test_get_git_context_non_git_dir(self):
+        """Test git context returns empty string for non-git directory."""
+        import importlib.util
+        hook_path = Path(__file__).parent.parent / "examples" / "notify-turn-hook.py"
+        spec = importlib.util.spec_from_file_location("hook", hook_path)
+        hook = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(hook)
+
+        # Use /tmp which is not a git repo
+        context = hook.get_git_context("/tmp")
```

Ruff (0.14.6)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)